### PR TITLE
Modal quiz results/ml

### PR DESCRIPTION
**Pull request by maddielaw**
## Purpose:

*What does this merge do? Check all that apply.*
- [X] New feature (non-breaking change which adds functionality)
- [X] Improves an existing feature
- [ ] Fixes a bug
      - Bug fixed:
- [ ] Refactor
      - Code refactored:
- [ ] Documentation update

## Description: What does this PR do?
This PR adds in a modal library to improve styling and UX for the quiz results.

## Checklist:
- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have used tests to prove my fix is effective or that my feature works
- [X] My changes generate no new warnings
- [ ] New and existing unit tests pass locally with my change

## Future Steps:
Polish and deploy!

### CODE REVIEW

**Comment by JessFatta** [comment]
Looks soo good! I love the styling for the quiz results and the little pop up tool tip for the state drop down! Great job!!

File: src/components/StateDropdown/StateDropdown.css [hunk 1]
```diff
@@ -3,8 +3,10 @@
   flex-direction: column;
   justify-content: center;
   align-items: center;
-  background-color: #457B9D;
-  border-radius: 15px;
+  border-top: 3px solid #457B9D;
+  border-bottom: 3px solid #457B9D;
+  /* background-color: #457B9D; */
```
Comment (zliibbe) [inline src/components/StateDropdown/StateDropdown.css:8]: Let's get these comments outta here. ☺️

**Review by nvalentini21 — APPROVED** [review]
The modal looks fantastic Maddie! All we have left is integrating our tests to reflect the new design, and we are good to go!

File: src/components/StateDropdown/StateDropdown.js [hunk 2]
```diff
@@ -77,21 +76,28 @@ const StateDropdown = (props) => {
...
           </select>
-            <button className='go-button' disabled={!props.selectedState}>Go!</button>
+          <Popup
+          trigger={<span><button className="go-button" disabled={!props.selectedState}>Go!</button></span>}
+          position="bottom center"
+          on={['hover', 'click', 'focus']}
+          closeOnDocumentClick
+          >
+            {!props.selectedState && <div className='dropdown-error-msg'>Please select a state to continue</div>}
+            {/* <button className='go-button' disabled={!props.selectedState}>Go!</button> */}
```
Comment (zliibbe) [inline src/components/StateDropdown/StateDropdown.js:96]: I think we can cut these comments out

**Review by zliibbe — APPROVED** [review]
I think this PR cleans up our view and the modal really adds an improved feel to the page. Thanks Maddie!

**Pull request merged**